netlify.toml

```diff
@@ -5,20 +5,20 @@
 
 # Standard Netlify redirects
 [[redirects]]
-    from = "https://kubernetes-sigs-cluster-api.netlify.com/*"
-    to = "https://cluster-api.sigs.k8s.io/:splat"
+    from = "https://master--kubernetes-sigs-cluster-api.netlify.com/*"
+    to = "https://dev.cluster-api.sigs.k8s.io/:splat"
     status = 301
     force = true
 
 # HTTP-to-HTTPS rules
 [[redirects]]
-    from = "http://go.cluster-api.sigs.k8s.io/*"
-    to = "https://go.cluster-api.sigs.k8s.io/:splat"
+    from = "http://dev.cluster-api.sigs.k8s.io/*"
+    to = "https://dev.cluster-api.sigs.k8s.io/:splat"
     status = 301
     force = true
 
 [[redirects]]
-    from = "http://kubernetes-sigs-cluster-api.netlify.com/*"
-    to = "http://cluster-api.sigs.k8s.io/:splat"
+    from = "http://master--kubernetes-sigs-cluster-api.netlify.com/*"
+    to = "http://dev.cluster-api.sigs.k8s.io/:splat"
```

Comment:
Would it save a redirect to make this https?

Reply:
Possibly, but I do know this is similar to the configuration we are using for the `release-0.2` branch and it is known to work :)